src/utils/api.js

```diff
@@ -0,0 +1,92 @@
+import {
+  getFirestore,
+  collection,
+  addDoc,
+  doc,
+  deleteDoc,
+} from "firebase/firestore";
+import { firebaseConfig } from "./firebaseConfig";
+import firebase from "firebase/compat/app";
```

Comment:
'compat' libraries are for upgrading legacy projects to firebase V9, they do not take advantage of the modular system savings. As we're starting with v9 we shouldn't need to use compats. 

current syntax: 
import { initializeApp } from 'firebase/app';